update.go

```diff
@@ -258,30 +264,34 @@ other options are ignored.
 		// Update the values
 		config.Cgroups.Resources.BlkioWeight = *r.BlockIO.Weight
 
-		// Setting CPU quota and period independently does not make much sense,
-		// but historically runc allowed it and this needs to be supported
-		// to not break compatibility.
+		// Setting CPU quota, period and burst independently does not make much
+		// sense, but historically runc allowed it and this needs to be
+		// supported to not break compatibility.
```

Comment:
This is historically true for period and quota, but not for burst. I think we should require period to be set when setting burst.

Reply:
Thanks, make sense to me. JFYI, underneath in kernel implementation, it looks like quota, period and burst are all set together. As for systemd, it's not yet supported for burst.